release-team/role-handbooks/docs/README.md

```diff
@@ -74,12 +106,102 @@ In addition to the time requirements above, a Docs Lead must:
 
+- Strong written and verbal communications skills
+- A working knowledge of Kubernetes concepts
+- Project management experience is helpful but not required
 - Have signed the [contributor CLA](https://github.com/kubernetes/community/blob/master/CLA.md) for Kubernetes.
 - Be invested in becoming an org member within the release cycle. This can often be achieved during the release cycle with sponsorship from a role lead. See the [Release Team onboarding guide](/release-team/release-team-onboarding.md) for more details.
-- General knowledge of our SIG-Docs [areas of responsibility](https://github.com/kubernetes/community/tree/master/sig-docs#subprojects).
+- General knowledge of our SIG Docs [areas of responsibility](https://github.com/kubernetes/community/tree/master/sig-docs#subprojects).
 - Experience with the general process involved with [contributing](https://kubernetes.io/docs/contribute/start/) to Kubernetes website.
 - Be a [milestone maintainer](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/) in order to have the ability to add a milestone to an issue. Access can be requested by creating a [PR](https://github.com/kubernetes/org/pull/2235) against `kubernetes/org` repo.
 
+### Machine and GitHub Setup
+
+#### Setup krel
+
+[Install Go](https://golang.org/doc/install) in your machine and follow the [instructions to build the release tools](https://github.com/kubernetes/release/tree/master/docs/krel#installation) in your machine. Check the system requirements in the krel documentation.
+
+#### Fork the kubernetes repositories
+
+Fork the following repositories to your GitHub account, and clone them using SSH:
+
+ - [`kubernetes/sig-release`](https://github.com/kubernetes/sig-release): This is where you will push regular PRs to keep the Release Notes draft up to date.
+ - [`kubernetes-sigs/release-notes`](https://github.com/kubernetes-sigs/release-notes): This repo has the [release notes website](https://relnotes.k8s.io) sources
+
 ## Release Timeline
 
 Reference the Docs [Release Timeline](Release-Timeline.md) for key dates and responsibilities during the release cycle and the Kubernetes Release Information page for the specific release (e.g. [Kubernetes 1.28 Release Information page](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.28/README.md) for information regarding the current release cycle including important dates, Release Team contact information, tracking spreadsheets, and more.
+
+## Tools
+- [krel](https://github.com/kubernetes/release/tree/master/docs/krel) The Kubernetes Release Toolbox *(note: always use the latest version of krel to ensure you have the latest fixes/patches)*
+- [The krel `release-notes` subcommand](https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md)
+- [The old release notes tool](https://github.com/kubernetes/release/tree/master/cmd/release-notes)
```

Comment:
Do we want to include a reference to the old release notes tool here? I don't think it's used by the release notes team anymore. If we want to include historical context, maybe we should link to Marko's talk on the release tooling: https://www.youtube.com/watch?v=ILpvdVDcSTU

Reply:
If it's no longer used at all, even in edge cases, it doesn't need to be included and I'll remove it. Thanks!

Reply:
Yep, it's not used by shadows creating the release notes so I think we can remove it.